stripe/src/main/java/com/stripe/android/StripeFireAndForgetRequestExecutor.kt

```diff
@@ -20,22 +27,24 @@ internal class StripeFireAndForgetRequestExecutor : FireAndForgetRequestExecutor
      */
     @VisibleForTesting
     @Throws(APIConnectionException::class, InvalidRequestException::class)
-    fun execute(request: StripeRequest): Int {
+    internal fun execute(request: StripeRequest): Int {
         // HttpURLConnection verifies SSL cert by default
         var conn: HttpURLConnection? = null
+        val responseCode: Int
         try {
             conn = connectionFactory.create(request)
 
             // required to trigger the request
-            val responseCode = conn.responseCode
+            responseCode = conn.responseCode
 
             closeConnection(conn, responseCode)
-            return responseCode
         } catch (e: IOException) {
             throw APIConnectionException.create(request.baseUrl, e)
         } finally {
             conn?.disconnect()
```

Comment:
if you wanted, you could have `connectionFactory.create()` return a `HttpURLConnection` wrapper class that implements `Closeable` , so you could simplify this try-catch-finally with `use {}`, but that's a nit, no big deal